### ISSUE TITLE
Better documentation

### ISSUE THREAD

**Issue by solarnz** [solarnz/gulp-invalidate-cloudfront#1]
We need some better documentation.

**Comment by jwulf** [comment]
Yep. How do I invalidate all the things in my distribution using this plugin?

**Comment by solarnz** [comment]
You shouldn't. You should version your assets instead, then only invalidate the files that reference those assets.

**Comment by builtbylane** [comment]
how do you invalidate a directory? currently I have `/example/index.html`
if I explicitly invalidate the index.html it doesn't seem to invalidate `http://foo.com/example`

**Comment by solarnz** [comment]
You can use wildcards, so you would invalidate `/example/*`

See http://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/Invalidation.html

**Comment by builtbylane** [comment]
I tried that, and it doesn't seem to work.
